deploy/sre-prometheus/100-managed-upgrade-operator.PrometheusRule.yaml

```diff
@@ -70,12 +70,12 @@ spec:
       annotations:
         summary: "Node drain failed in the given time period which is not caused by the PDB"
         description: "node drain takes too long and cannot be finished in the given time period during cluster upgrade"
-    - alert: UpgradeConfigSyncFailureOver1HrSRE
-      expr: upgradeoperator_upgradeconfig_synced > 0
+    - alert: UpgradeConfigSyncFailureOver2HrSRE
+      expr: avg_over_time(upgradeoperator_upgradeconfig_synced[2h]) == 1
```

Comment:
This looks good, I like the change.
How does the `for` (line below) factor in to this when using `avg_over_time` though - am I right in thinking that this can be dropped to something like `5m` rather than `1h` (if it is even necessary at all..?)?

Reply:
good catch, yes, we do not need 1 hour more here, I will check it to 5 minutes to fire the alert